test/zlib/test_zlib.rb

```diff
@@ -12,7 +12,18 @@
 end
 
 if defined? Zlib
-  class TestZlibDeflate < Test::Unit::TestCase
+  class Zlib::TestCase < Test::Unit::TestCase
+    def child_env
```

Comment:
The `child_env` passed to subprocesses is not modified, correct?  If so, I recommend this be a frozen constant and not a method.  You could define it as `Zlib::CHILD_ENV`, and skip creating a subclass and modifying the tests to use the subclass.

Reply:
Yes, the `child_env` is not modified. Sure. Let me modify the PR on your way.

Reply:
I rebased fixing it on your way. Now what do you think?